frontend/common/stores/user-group-store.js

```diff
@@ -50,7 +50,7 @@ const controller = {
       { orgId: `${orgId}`, page: 1 },
       { forceRefetch: true },
     ).then((response) => {
-      store.groups = response[0].data.results
+      store.groups = response.data.results
```

Comment:
No longer necessary and wouldn't have worked well. GetGroups now just returns its own data